packages/fuel-indexer-macros/src/schema.rs

```diff
@@ -59,13 +60,30 @@ fn process_field<'a>(
     let typ = process_type(types, field_type, true);
     let ident = format_ident! {"{}", name};
 
-    let extractor = quote! {
-        let item = vec.pop().expect("Missing item in row");
-        let #ident = match item {
-            FtColumn::#typ(t) => t,
-            _ => panic!("Invalid column type {:?}", item),
-        };
+    // Type may be nullable, so let's grab the actual column type
```

Comment:
@deekerno We can remove these types of comments. They don't add any real information beyond how the code reads